cross/curl/Makefile

```diff
@@ -5,16 +5,19 @@ PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = https://curl.se/download
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
-DEPENDS = cross/zlib cross/openssl
+DEPENDS = cross/zlib cross/openssl cross/gnutls cross/libssh2 cross/zstd cross/libwebsockets
```

Comment:
@th0ma7 is this really required?
IMHO it makes no sense to have two TLS libraries as default in curl (openssl and gnutls). 
This blows up the curl package (I already tried to minimize it, by adding the curl libraries by default and make the curl executable optional).

Asking, as I am building an update for git, while it fails now for arch-ppc853x-5.2, since gnutls >= 3.7.4 is in the dependencies.

if tvh _really_ requires to build libcurl with gnutls, libssh2 zstd and libwebsockets and curl build does not autodetect such dependencies, we need to enhance cross/curl for advances configuration (as we do for cross/busibox, cross/pcre2, et al.)

What's your opinion?

Reply:
we could probably play with the build order to ensure gnutls gets built before curl and therefore allowing it to autodetecting it and link to it without having to specifying it.  As such a normal curl build would no make use of it.  Would that make sense?